components/math/multiplicationConnect/GameBoard.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+const GameBoard = () => {
+  return (
+    <div className="px-28">
+      <h2 className='text-xl pb-4'>GameBoard</h2>
+      <div className="grid grid-cols-5 border-t-2 border-l-2 border-t-blue-600 border-l-blue-600 bg-blue-900">
+        <div className='flex justify-center items-center border-r-2 border-b-2 h-28 border-b-blue-600 border-r-blue-600 bg-blue-900'>1</div>
```

Comment:
Can we clean this up by creating an array of numbers, then mapping through that array to render these divs?

Reply:
Good idea, thank you!